### PR TITLE
Update ReadMe for gen2

### PR DESCRIPTION
**Pull request by ardiya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,14 +10,13 @@ This meta-package contains the following packages:
 # Setup
 ## Requirements
 Though this metapackage might work on other systems, it has only been tested on:
-* Ubuntu 18.04 LTS
```
Comment (ardiya) [inline README.md:13]: Remove Ubuntu 18 since it doesn't have OpenCV 4

File: README.md [hunk 2]
```diff
@@ -10,14 +10,13 @@ This meta-package contains the following packages:
...
 * Ubuntu 20.04 LTS
 
 A list of pre-requisites:
 * C++ compiler (recommended: `g++` or `clang` with at least C++17 support)
 * CMake (3.10.2 or higher)
 * Boost (1.65.1 or higher)
-* OpenCV (3.2 or higher)
+* OpenCV (4.0 or higher)
```
Comment (ardiya) [inline README.md:19]: `depthai-core` needs OpenCV 4

File: README.md [hunk 3]
```diff
@@ -30,42 +29,25 @@ For older versions of git, one of the following processes might be needed:
 * Use `--recurse-submodules` instead of `--recursive`
 * Clone without any flags and then grab the submodules using `git submodule update --init --recursive`
 
-## Grab the Neural Network Model
-* Prepare MobileNetSSD compatible model (blob and json file). Once issue #5 is resolved, this will not be necessary for a default model. Due to limitations of Gen1 API used in the ROS driver, a model is required to run it.
-
-Sample models are available [here](https://github.com/luxonis/depthai/tree/main/resources/nn)
-
```
Comment (ardiya) [inline README.md]: blob already provided? not needed?

File: README.md [hunk 4]
```diff
@@ -30,42 +29,25 @@ For older versions of git, one of the following processes might be needed:
...
 # Usage
 0. Compile using `catkin build depthai_ros` or `catkin_make_isolated --build depthai_ros`. `catkin_make` will **not build** the package successfully.
 1. Execute `./deploy_udev_rules.sh` present in `~/depthai_ros/depthai_ros_driver/scripts`.
 2. Launch the node or nodelet by appropriately choosing the correct launch file and parameters:
     ```
     roslaunch depthai_ros_driver depthai_{node,nodelet}.launch
     ```
-3. Custom models can be provided using `blob_file` and `blob_file_config` arguments to the launch file. If an argument `A` needs value `B`, the syntax is `roslaunch <launch_file> A:=B`
```
Comment (ardiya) [inline README.md:45]: `blob_file_config` not needed

**Review by kunaltyagi — DISMISSED** [review]

File: README.md [hunk 5]
```diff
@@ -30,42 +29,25 @@ For older versions of git, one of the following processes might be needed:
...
-```
-Stream not opened: config_d2h
-Available streams: []
-device is not initialized
-Pipeline is not created.
-No USB device [03e7:2485], still looking... 0.401s [FOUND]
-Sending internal device firmware
-Successfully connected to device.
-```
-Try forcing USB2 mode using `<roslaunch as before> force_usb2:=true`
```
Comment (ardiya) [inline README.md:64]: USB2 runs out of the box?

File: README.md (hunk 3 above)
Comment (kunaltyagi) [reply README.md]: Needed for other NN

Comment (ardiya) [reply README.md]: reverted with updated sample models

**Pull request closed without merging**